Makefile

```diff
@@ -517,7 +521,41 @@ $(BUILD_DIR)/profiles.ttl: $(BUILD_DIR)/evolutionary-profiles.ttl $(BUILD_DIR)/g
 
 # ##########
 
+# ########## # ##########
+# ########## # ##########
+
+
+
+
+
+# ########## # ##########
+# ########## # ##########
+
+
+# ##########
+# Load pipeline artifacts into Blazegraph database
+
+# Artifacts:
+# 1. Phenoscape KB
+# 2. Closures = subclass + instance)
+# 3. taxa-sim = taxa-ics + taxa-expect-scores + taxa-pairwise-sim
+# 4. gene-sim = gene-ics + gene-expect-scores + gene-pairwise-sim
+
+$(BUILD_DIR)/blazegraph-loaded-all.jnl: $(BUILD_DIR)/phenoscape-kb.ttl \
+										$(BUILD_DIR)/subclass-closure.ttl $(BUILD_DIR)/instance-closure.ttl \
+										$(BUILD_DIR)/corpus-ics-taxa.ttl $(BUILD_DIR)/taxa-expect-scores.ttl $(BUILD_DIR)/taxa-pairwise-sim.ttl \
+										$(BUILD_DIR)/corpus-ics-genes.ttl $(BUILD_DIR)/gene-expect-scores.ttl $(BUILD_DIR)/gene-pairwise-sim.ttl
```

Comment:
I would `rm` any existing database file before starting. Otherwise you could mix old and new content.